packages/zipkin/rollup.config.js

```diff
@@ -5,6 +5,7 @@ import replace from 'rollup-plugin-replace';
 import {terser} from 'rollup-plugin-terser';
 import globals from 'rollup-plugin-node-globals';
 import builtins from 'rollup-plugin-node-builtins';
+import del from 'rollup-plugin-delete';
```

Comment:
^ This is the plugin doing the trick for the clean up of old build files.

Reply:
cool